portal/static/js/main.js

```diff
@@ -355,11 +356,12 @@ var ConsentUIHelper = function(consentItems, userId) {
         };
         //Note: Truenth and Eproms have different text content for each column.  Using css classes to hide/show appropriate content
         //wording is not spec'd out for EPROMs. won't add anything specific until instructed
+
         this.touObj.forEach(function(item, index) {
             var org = OT.orgsList[item.organization_id];
             touContent += "<tr data-tou-type='" + item.type + "'>";
             touContent += "<td><span class='eproms-tou-table-text'>" + (org && hasValue(org.name) ? i18next.t(org.name) : "--") + "</span><span class='truenth-tou-table-text'>TrueNTH USA</span></td>";
-            touContent += "<td><span class='text-success small-text eproms-tou-table-text'>Agreed to <a href='" + item.agreement_url + "' target='_blank'><span class='text-capitalize'>" + i18next.t(item.display_type) + "</span></a></span><span class='text-success small-text truenth-tou-table-text'>" + i18next.t("Agreed to terms") + "</span></td>";
+            touContent += "<td><span class='text-success small-text eproms-tou-table-text'><a href='" + item.agreement_url + "' target='_blank'>" + i18next.t("Agreed to " + capitalize(item.display_type)) + "</a></span><span class='text-success small-text truenth-tou-table-text'>" + i18next.t("Agreed to terms") + "</span></td>";
```

Comment:
I don't think you can pass dynamically-generated strings to `i18next.t()` without string formatting/interpolation. 

`capitalize()` won't be called until after `i18next-extraction` is run, so `item.display_type` won't be included in the POT file. Using interpolation allows you to substitute the value in *after* the string is translated by `i18next.t()`, so it should appear correctly in the translated text.